src/js/core/config.js

```diff
@@ -1,5 +1,4 @@
 /* typehints:start */
-import { Application } from "../application";
 /* typehints:end */
```

Comment:
empty typehints block?

Reply:
I have automatic import optimization turned on, I will resolve this in a further commit to this PR